xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -134,6 +134,7 @@ bool CRenderManager::Configure(const VideoPicture& picture, float fps, unsigned
     m_fps = fps;
     m_orientation = orientation;
     m_stereomode = picture.stereoMode;
+    m_overlays.SetStereoMode(m_stereomode);
```

Comment:
this is wrong Configure method. I meant the one without arguments. Thats why I explicitly wrote "Configure()"